spec/features/users/profile_recovery_for_gpo_verified_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.feature 'Password recovery via personal key for a GPO-verified user' do
+  include IdvStepHelper
+
+  let(:email) { 'cool_beagle@test.org' }
```

Comment:
minor nit: can we make this @example.com?

Reply:
Done